pyqt-apps/siriushla/as_ap_sofb/ioc_control/respmat.py

```diff
@@ -30,6 +30,12 @@ class RespMatWidget(BaseWidget):
 
     def __init__(self, parent, device, prefix='', acc='SI'):
         super().__init__(parent, device, prefix=prefix, acc=acc)
+        self._enblrule = (
```

Comment:
@anacso17, you reuse this rule several times along the code. Isn't there a way to avoid keeping redefining it?

Reply:
Hi @fernandohds564 , do you think it is useful to define this rule in a base class? Maybe a "base" rule, to which we could pass a channel?

Reply:
maybe, or have some variable defined in utils...

Reply:
never mind @anacso17. I already made a PR on top of this one. We can change this later.